docs/concepts/flatcar-container-linux.md

```diff
@@ -0,0 +1,83 @@
+# Operating system
+
+## Flatcar Container Linux
+
+Lokomotive installs [Flatcar Container Linux](https://www.flatcar-linux.org/) as the underlying operating system.
+
+Flatcar Container Linux is an open source immutable Linux distribution for containers. It is a
+friendly fork of CoreOS Container Linux and as such, compatible with it.
+
+### Why Flatcar Container Linux ?
+
+* Minimal distribution required for containers.
+  * Reduced dependencies.
+  * Reduced attack surface area.
+* Immutable file system.
+  * Operational simplicity.
+  * Removes entire category of security threats. (e.g. runc vulnerability CVE-2019-5736)
+    https://kinvolk.io/blog/2019/02/runc-breakout-vulnerability-mitigated-on-flatcar-linux/
+* Automated, streamlined and atomic updates.
+  * Easily apply all latest security patches.
+  * Rollback partition.
+* Declarative and immutable configuration.
+* Optimization for containerized applications.
+
+### Properties
+
+Flatcar Container Linux specific properties.
+
+### Host properties
+
+| Property               | Flatcar Container Linux |
+|------------------------|-------------------------|
+| host spec (bare-metal) | Container Linux Config  |
+| host spec (cloud)      | Container Linux Config  |
+| container runtime      | docker                  |
+| cgroup driver          | cgroupfs                |
+| logging driver         | json-file               |
+| storage driver         | overlay2                |
```

Comment:
These are properties of the Docker container runtime so it's weird that they're in the same level as Docker. Maybe this whole section can be reworked as follows:

```
Flatcar Container Linux uses Docker as its container runtime. This is the default configuration:

| Setting | Value |
| --------- | --------- |
| cgroup driver | cgroupfs
...